hackathon_site/hardware/api_filters.py

```diff
@@ -0,0 +1,25 @@
+from django_filters import rest_framework as filters
+from django_filters.fields import Lookup
+
+from hardware.models import Hardware
+from hardware.serializers import HardwareSerializer
+
+
+class HardwareFilter(filters.FilterSet):
+    queryset = Hardware
+    serializer_class = HardwareSerializer
+
+    class Meta:
```

Comment:
We actually don't need this meta block at all.

If you look at swagger for this view (http://localhost:8000/swagger/#/hardware/hardware_hardware_list), this adds two extra fields for name and quantity available. We don't need a filter field for name (since it's covered by the search box), and we don't need a filter for quantity_available either (the in stock filter is all we need there). So this meta block can be removed methinks